Universal/usbw-methods/manual.md

```diff
@@ -0,0 +1,19 @@
+# Fixing USB Wake: Manual
+
+* [Grabbing the sample SSDT](#grabbing-the-sample-ssdt)
+* [Compiling the SSDT](#compiling-the-ssdt)
+* [Wrapping up](#wrapping-up)
+
+## Grabbing the sample SSDT
+
+[Download SSDT-USBW.dsl here.](https://raw.githubusercontent.com/osy/USBWakeFixup/master/SSDT-USBW.dsl)
+
+## Compiling the SSDT
+
+This SSDT requires no modification, so you're now [ready to compile the SSDT!](/Manual/compile.md)
```

Comment:
If no modifications are required, can we just have a single page which links both the manual and compiled version? Seems a little weird to say "You don't really learn anything" when you don't edit the file (or even open it lol).

Reply:
Hah, yeah that is a bit weird. That bit about it "not teaching you anything" is copied verbatim from the other SSDT guides, I would have taken it out if I'd thought about that :stuck_out_tongue: See if you like the changes in 5ef772a.

Reply:
Looks good